code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -2,9 +2,9 @@
 	name = "clutter"
 	desc = "Someone should clean that up."
 	gender = PLURAL
-	density = 0
-	anchored = 1
-	layer = 2
+	density = FALSE
+	anchored = TRUE
+	layer = 2.5
```

Comment:
hm, maybe you could define layer = 2.5 as ABOVE_PIPE_LAYER 

or at least define it as ABOVE_PIPE_LAYER (PIPE_LAYER + 0.1)
since we already have PIPE_LAYER at 2.4

Reply:
Yeah, no magic numbers here please